.circleci/config.yml

```diff
@@ -425,11 +425,13 @@ jobs:
           test_targets: "asm*.test_j* asm*.test_k* asm*.test_l* asm*.test_m* asm*.test_n* asm*.test_o*"
   test-fastcomp-p:
     executor: bionic
+    environment:
+      # Without this test_poppler can cause timeouts because it takes more then
+      # 10 minutes without outputing anything.
+      EM_BUILD_VERBOSE: "2"
```

Comment:
this affects all tests in this suite, but I guess there aren't many in it?

did you notice if adding this makes the build slower (while making it more robust to timeouts)?